### PR TITLE
Proofread the first half of chapter seven

### PR DESCRIPTION
**Pull request by KRS-Newberg**
I tried to execute the command given in github-requests but didn't get any output. here's hoping it worked alright!

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_1B.txt [hunk 1]
```diff
@@ -245,7 +245,7 @@ In truth, Noel-sensei's joke hit the nail right on the head,
 [sha:53b66a730f0b41e286911913f90f88db8fdb648b]{
 -- Page 56, Offset 16715.
 -- 「貧血の方はなんとか。今朝は特に調子がいいです」
-"For someone with anemia, I feel in pretty good shape this morning."
+"For someone with anemia, I am in pretty good shape this morning."
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_1B.txt:248]: positive change, but swap to contraction. This doesn't strike me as a place for emphasis

File: script/Arcueid/Day 7/07_00_ARC07_1B.txt [hunk 2]
```diff
@@ -537,7 +537,7 @@ The burglars are getting caught before the police even have a chance to show up,
 [sha:47f32f5617b1cd30ab111db1c69f5ca6aa841b2f]{
 -- Page 78, Offset 16766.
 -- 「なるほど。それならどんな噂も広まらないか」
-"I see. In that case, there'd be no way for rumors to spread."
+"I see. If that's the case, then that'd be why no rumors have spread."
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_1B.txt:540]: The "i see." here is so stiff. Maybe soften it and replace それ with the actual thing hes referring to
"Oh, I get it. If they can't get out, then there's no way for rumors to spread."

File: script/Arcueid/Day 7/07_00_ARC07_1C.txt [hunk 3]
```diff
@@ -267,7 +267,7 @@ So then why... Why are corpses with their blood sucked dry still appearing...?!
 -- Page 109, Offset 16841. Glued. Mods: @x.
 -- 相手が美人のお姉さんなら吸われちゃってもいいんだがねぇ」
 // Leading space for glue
- Although I guess I wouldn't mind having my blood sucked if the vampire was a beautiful older woman."
+ Although I guess I wouldn't mind having my blood sucked if the vampire was a beautiful onee-san."
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_1C.txt:270]: I think revert this change and the next
I'm for honorifics being maintained because they encode extra information which is missing in English. I don't think there's anything gained in this case.

File: script/Arcueid/Day 7/07_00_ARC07_2_2.txt [hunk 4]
```diff
@@ -34,6 +34,7 @@ Akiha's anger, having swollen like an inflating balloon,
 [sha:dfb27e28ee76a513a409b2bde68d4506820b564a]{
 -- Page 166, Offset 16982.
 -- 「……ちょっと頭いたい。自称吸血鬼だって言い張る女より、うちの妹の方が迫力あるとか……」
+//I don't know what to do about this line but I am suspect of 'Self proclaimed' and 'Doing my head in'
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_2_2.txt:37]: This line is very weird and during the various TLCs we've discussed it quite thoroughly. As for the first part:
> ちょっと頭いたい
lit. "This kinda hurts my head."

and for the second:
> 自称吸血鬼だって言い張る女
lit. The woman who insists shes a self-proclaimed vampire

Which is kinda rough. This line is more-or-less accurate.

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 7/07_00_ARC07_1C.txt (hunk 3 above)
Comment (KRS-Newberg) [reply script/Arcueid/Day 7/07_00_ARC07_1C.txt:270]: I really don't like the translation of 'onee-san' as 'older-woman' since that has a much different (to my knowlage) connotation in english. Given our audience is people who read visual novels I think that 'onee-san' is the better choice.

Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_1C.txt:271]: I might side with Ben on this one, I think there are a couple times Arihiko's preference for a more mature type comes up - 
```
Day 3/03_00_ARC03_3.txt-[sha:d6281cfd00da2f05ac19f0869912a3074900faf2]{
Day 3/03_00_ARC03_3.txt--- Page 162, Offset 4362.
Day 3/03_00_ARC03_3.txt--- 「眼鏡の似合う上級生だって言うんだろ？」
Day 3/03_00_ARC03_3.txt:"You're saying you like older women who look good in glasses, right?"
Day 3/03_00_ARC03_3.txt-}
```
I think it's referenced again re: Noel when taking to Arihiko, but I can't find the line

**Review by rschlaikjer — APPROVED** [review]

File: script/Arcueid/Day 7/07_00_ARC07_1C.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_1C.txt:271]: Ah, here it is
```
_ 635 -- おまえ、包容力のある年上が好きなんだろ？　良かったな、好み通りの25歳だ」
  636  You like older women who are patient, don't you? Good for you. She's 25, exactly how you like it."
```
